comp/core/workloadmeta/collectors/internal/kubeapiserver/deployments.go

```diff
@@ -22,11 +21,11 @@ import (
 
 	"github.com/DataDog/datadog-agent/comp/core/workloadmeta"
 	"github.com/DataDog/datadog-agent/pkg/languagedetection/languagemodels"
+	languagedetectionUtil "github.com/DataDog/datadog-agent/pkg/languagedetection/util"
+
 	ddkube "github.com/DataDog/datadog-agent/pkg/util/kubernetes"
 )
 
-var re = regexp.MustCompile(`apm\.datadoghq\.com\/(init)?\.?(.+?)\.languages`)
```

Comment:
We can reuse the same regex initialised in the util package of language detection